sdk/trace/src/testIncubating/java/io/opentelemetry/sdk/trace/TracerConfigTest.java

```diff
@@ -42,9 +41,13 @@ void disableScopes() throws InterruptedException {
             .addSpanProcessor(SimpleSpanProcessor.create(exporter))
             .build();
 
-    Tracer tracerA = tracerProvider.get("tracerA");
-    Tracer tracerB = tracerProvider.get("tracerB");
-    Tracer tracerC = tracerProvider.get("tracerC");
+    ExtendedSdkTracer tracerA = (ExtendedSdkTracer) tracerProvider.get("tracerA");
+    ExtendedSdkTracer tracerB = (ExtendedSdkTracer) tracerProvider.get("tracerB");
+    ExtendedSdkTracer tracerC = (ExtendedSdkTracer) tracerProvider.get("tracerC");
+
+    assertThat(tracerA.isEnabled()).isTrue();
+    assertThat(tracerB.isEnabled()).isFalse();
+    assertThat(tracerC.isEnabled()).isTrue();
```

Comment:
did you want to keep the same asserts at the end of this test?

Reply:
This test is all about static behavior, so I don't think we need to verify the state is unchanged at the end

Reply:
I'd add a comment there in the PR if github let me, this is what I'm referring to at the bottom of this test:

```
    // tracerA and tracerC are enabled, tracerB is disabled.
    assertThat(((ExtendedTracer) tracerA).isEnabled()).isTrue();
    assertThat(((ExtendedTracer) tracerB).isEnabled()).isFalse();
    assertThat(((ExtendedTracer) tracerA).isEnabled()).isTrue();
```

Reply:
Ah! missed that this was already asserted. Fixed. Thanks.